idunn/geocoder/models/params.py

```diff
@@ -30,54 +30,70 @@ class Type(str, Enum):
     Zone = "zone"
 
 
+class PlaceDocType(str, Enum):
+    # pylint: disable=invalid-name
+    Admin = "admin"
+    Street = "street"
+    Addr = "addr"
+    Poi = "poi"
+    Stop = "stop"
```

Comment:
I think Idunn should not be aware about `stop` type

Reply:
This makes sense :+1:

Reply:
Do you think we should also remove `StopArea = "public_transport:stop_area"` from the enum `Type`?

Reply:
yes